src/mbgl/util/mapbox.cpp

```diff
@@ -85,6 +95,11 @@ std::string normalizeGlyphsURL(const std::string& url, const std::string& access
     }
 
     std::vector<std::string> pathname = getMapboxURLPathname(url);
+
+    if (pathname.size() < 3) {
```

Comment:
This is in the wrong section: I've adjusted the code to < 4 for the glyph endpoint